install_config/persistent_storage/persistent_storage_ceph_rbd.adoc

```diff
@@ -11,21 +11,27 @@
 toc::[]
 
 == Overview
-OpenShift can utilize persistent storage using Block Storage
-like Ceph RBD. Some familiarity with Kubernetes and Docker is assumed. It is
-also assumed that there is access to an existing Ceph cluster and volume,
-and that *ceph-common* package has been installed on all OpenShift nodes in the
-cluster.
-
-The Kubernetes
-link:../../architecture/additional_concepts/storage.html[persistent volume]
-framework allows administrators to provision a cluster with persistent storage
-and gives users a way to request those resources without having any knowledge of
-the underlying infrastructure. Persistent volumes are not bound to a single
-project or namespace; they can be shared across the OpenShift cluster.
-link:../../architecture/additional_concepts/storage.html#persistent-volume-claims[Persistent
-volume claims], however, are specific to a project or namespace and can be
-requested by users.
+
+OpenShift clusters can be provisioned with
+link:../../architecture/additional_concepts/storage.html[persistent block storage]
+such as Ceph-RBD. Some familiarity with OpenShift,
+link:../../architecture/additional_concepts/storage.html#persistent-volume-claims[persistent
+volume claims (PVCs)], and with
+https://access.redhat.com/products/red-hat-ceph-storage[Ceph] is beneficial.
+
+As a very brief review, the OpenShift persistent volume (PV) framework allows administrators to
```

Comment:
@pmorie @jeffvance Yeah, see http://mrhaki.blogspot.com/2014/04/awesome-asciidoc-include-partial-parts.html. Example in our docs:

https://github.com/openshift/openshift-docs/blame/master/cli_reference/get_started_cli.adoc#L306

pulls in...

https://github.com/openshift/openshift-docs/blame/master/dev_guide/authentication.adoc#L118-L135